test/functional/api/cas/casadm.py

```diff
@@ -17,6 +17,8 @@
 from .casadm_params import *
 from .casctl import stop as casctl_stop
 from .cli import *
+from storage_devices.lvm import Lvm
+from test_utils.disk_finder import resolve_to_by_id_link
```

Comment:
Those two lines are added in the "lvm api (ocl)" commit and removed in the next one "lvm common: improve get_test_configuration function". No need to add them in the first place, especially that those imports are not used anywhere.